doc/api/repl.markdown

```diff
@@ -106,6 +108,14 @@ the following values:
 You can use your own `eval` function if it has following signature:
 
     function eval(cmd, context, filename, callback) {
+      var result;
+      try {
+        result = vm.runInThisContext(cmd);
+      } catch (e) {
+        if (isRecoverableError(e)) {
```

Comment:
> So, lets remove that part alone.

What do you mean by this? Shouldn't this be removed all together, the following regexp test is pretty clear.